pkg/exchange/okex/kline_stream.go

```diff
@@ -32,14 +34,24 @@ func NewKLineStream() *KLineStream {
 	return k
 }
 
+func (s *KLineStream) UseFutures() {
+	s.isFutures = true
+}
+
 func (s *KLineStream) handleConnect() {
 	var subs []WebsocketSubscription
+
+	instType := okexapi.InstrumentTypeSpot
+	if s.isFutures {
+		instType = okexapi.InstrumentTypeSwap
+	}
```

Comment:
see my PR to call the new method to get the instrument type
https://github.com/c9s/bbgo/pull/2023/files#diff-14a80c286a74b4b35bf3fa52f7e522f1db4e9c7f13ce1706b64b95547b193e5eR805